lib/assessment.js

```diff
@@ -4,8 +4,9 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-var min = function min() {
-
+var min = function min(ary) {
+  ary = ary.sort();
+  return ary[0];
 };
```

Comment:
This will work great when passed an array of numbers as the first argument. The question asked for a function that takes an indeterminate number of arguments, though; that suggests using the `arguments` pseudo-array. 3/4 credit.